enginetest/queries/queries.go

```diff
@@ -1718,6 +1718,15 @@ Select * from (
 			{types.MustJSON(`["d"]`)},
 		},
 	},
+	{
+		Query: "SELECT JSON_SEARCH(c3, 'all', '2') FROM jsontable;",
+		Expected: []sql.Row{
+			{nil},
+			{nil},
+			{nil},
+			{nil},
+		},
```

Comment:
(minor) This could be a little stronger of a test if some of the expected values weren't `nil`. That way, it'd exercise the code path in `json_search` that returns results, too. I see you have that covered at a high level in `script_queries.go` already, so maybe this test query isn't needed here?

Reply:
removed this test in favor of the script tests